src/web/app/pages-instructor/instructor-course-student-details-page/instructor-course-student-details-page.component.ts

```diff
@@ -22,8 +22,13 @@ export class InstructorCourseStudentDetailsPageComponent implements OnInit {
   studentProfile?: StudentProfile;
   photoUrl: string = '';
 
+  courseId: string = '';
```

Comment:
Any specific reason to promote `courseId` to class attribute?

Reply:
This was so that I could call the `loadStudentDetails` method again